spec/models/occupation_standard_spec.rb

```diff
@@ -82,6 +82,26 @@
     end
   end
 
+  describe ".by_state_abbreviation" do
+    it "returns records that have a registration agency for that state abbreviation" do
+      ca = create(:state, abbreviation: "CA")
+      wa = create(:state, abbreviation: "WA")
+      ra_ca = create(:registration_agency, state: ca)
+      ra_wa = create(:registration_agency, state: wa)
+      os1 = create(:occupation_standard, registration_agency: ra_ca)
+      os2 = create(:occupation_standard, registration_agency: ra_ca)
```

Comment:
I found a little bit hard to read this at first with all the abbreviations. Can we optimize for clarity a bit? We could then prefer to use state names that are shorter and easier to type

```suggestion
      agency_california = create(:registration_agency, state: california)
      agency_utah = create(:registration_agency, state: utah)
      standard1 = create(:occupation_standard, registration_agency: agency_california)
      standard2 = create(:occupation_standard, registration_agency: agency_california)
      create(:occupation_standard, registration_agency: agency_utah)
```